docs/PATCH-ENVELOPES.md

```diff
@@ -1,36 +1,99 @@
 # Patch Envelopes
 
-## Overview
+## What is Patch Envelope? 
 
-Patch Envelopes are objects which are exposed to app instances via EVE meta-data server.
-This objects can be useful to update any kind of information in a secure, isolated manner
-(for instance, configuration parameters) on app instance without the need of rebooting it.
-Alternatively, one can create new image for app instance, upload it to EVE, purge and restart
-application instance with the new image, even if there was a small change in some configuration
-file. To top that there will be a down-time during reboot of app instance. To summarise it,
-patch envelopes goal is to make fleet management of app instances easier.
+Patch Envelope is a functionality in EVE that helps you change data in your App Instance in
+runtime without the need to reboot instance, or recreate image. Imagine that in order to change
+configuration in one of your program, you have to create new OS image and flush it to your computer.
+With Patch Envelope you don’t have to do that. Patch Envelope constitutes of Binary Artefacts (blobs)
+which are base64-encoded objects this can be anything, from binary file to a configuration yaml file.
```

Comment:
Should "this can be anything..." be a new sentence?

Reply:
Done